Jellyfin.Data/Enums/VideoRangeType.cs

```diff
@@ -26,10 +26,25 @@ public enum VideoRangeType
     HLG,
 
     /// <summary>
-    /// Dolby Vision video range type (12bit).
+    /// Dolby Vision video range type (10bit).
```

Comment:
Profile 5 uses ICtCp instead of YCbCr, which is more efficient. So it's 10-Bit encoded/12-bit remapped.